frontend-react/docs/standard-operating-procedures/e2e-frontend-guidance.md

```diff
@@ -32,6 +32,14 @@ If your e2e tests fail during the GitHub build. The following steps will help in
 
 ---
 
+## Updating the TEST_*_USERNAME OKTA accounts
+
+1. Change the password in OKTA for the `TEST_*_USERNAME` user account(s).
+2. Have DevOps update the GitHub Action `TEST_*_PASSWORD` secret(s) for the `TEST_*_USERNAME` user account(s).
+3. Have DevOps update the Dependabot `TEST_*_PASSWORD` secret(s) in the Security tab for the `TEST_*_USERNAME` user account(s).
```

Comment:
Both Action secrets and Dependabot secrets are under `Settings`/`Secrets and variables`. You might not be able to see that.  There is no **Security tab.**
The direct URLs are:
https://github.com/CDCgov/prime-reportstream/settings/secrets/actions
https://github.com/CDCgov/prime-reportstream/settings/secrets/dependabot